test/tests/functional/pbs_multi_sched.py

```diff
@@ -393,7 +393,9 @@ def test_preemption_highp_queue(self):
         j = Job(TEST_USER1, attrs={ATTR_queue: 'wq4',
                                    'Resource_List.select': '1:ncpus=2'})
         jid5 = self.server.submit(j)
-        self.server.expect(JOB, {'job_state': 'Q'}, id=jid5)
+        job_c = 'Not Running: No available resources on nodes'
+        self.server.expect(JOB, {'comment': job_c, 'job_state': 'Q'},
+                           attrop=PTL_AND, id=jid5)
```

Comment:
No need to check for what the comment is set to.  We don't care.  We just need to check that it is set.  By the way, this is a very generic comment message.  If things are going well, we should never see it.  It's kind of a last ditch effort of giving a message back.  This probably should be looked into.

Reply:
Changed to just checking comment is set